pkg/kubelet/dockertools/container_gc.go

```diff
@@ -245,35 +281,56 @@ func (cgc *containerGC) GarbageCollect(gcPolicy kubecontainer.ContainerGCPolicy,
 	return nil
 }
 
-func (cgc *containerGC) removeContainer(id string, podNameWithNamespace string, containerName string) {
-	glog.V(4).Infof("Removing container %q name %q", id, containerName)
-	err := cgc.client.RemoveContainer(id, dockertypes.ContainerRemoveOptions{RemoveVolumes: true})
+func (cgc *containerGC) netContainerCleanup(containerInfo containerGCInfo) {
```

Comment:
@caseydavenport @feiskyer I agree with Casey, killing a dead infra container during sync is an exceptional event, not a normal one.  So I think that would be OK.